models/Users.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require('mongoose');
+
+const UserSchema = new mongoose.Schema({
+  
+    name:{
+        type:String,
+        required:true
+    },
+    email:{
+        type:String,
+        required:true,
+        unique:true
+    },
+    username:{
+        type:String,
+        required:true,
+        unique: true
+    },
+    password:{
+        type:String,
+        required:true
+    },
+
+    date:{
```

Comment:
What does `date` signify? The day the record was created? The day it was modified?

Reply:
Yes

Reply:
Try to be more specific with your names, like 'created_at', 'updated_at'

You can lookup the 'timestamps' schema option, it might help